configs/components/oifs/oifs.yaml

```diff
@@ -696,7 +704,7 @@ choose_with_xios:
                                         LXIOS: ".true."
                 outdata_files:
                         oifsnc: oifsnc
-                        atm: atm
+                        #atm: atm
```

Comment:
I believe this is important for `AWICM3` and it shouldn't be removed. Is it giving any problem to `oifs` standalone or `focioifs`?

Reply:
@joakimkjellsson can you comment on this. I added it again, did a short test, and everything seems to be fine. I'll add a commit where the `#`is removed.